src/bin/release-please.ts

```diff
@@ -126,6 +126,14 @@ const argv = yargs
           describe: 'include library name in tags and release branches',
           type: 'boolean',
           default: false,
+        })
+        .option('draft', {
```

Comment:
@bcoe how do you feel about this cli option name? I think the release-please-action input should be `create-release-as-draft` as you suggested but that felt repetitive here. However, I can also see the argument for consistency.

Reply:
Why don't we add [an alias](https://github.com/yargs/yargs/blob/master/docs/api.md#optionskey-opt), `create-release-as-draft`, but keep the shorthand too.

In the GitHub Action, we can document the long form arguments, but for folks experimenting with the CLI, save a few key strokes.

Reply:
got a little confused about this suggestion in combination with your **{pr,release}-foo** prefixing suggestions below (which I like). Assuming that suggestion was for the cli (as opposed to the action) should I

- [ ] add an alias `create-release-as-draft`
- [ ] add an alias `release-create-as-draft`
- [ ] add aliases for options that only pertain to one command: (i.e. `pr-default-branch`, `pr-version-file`, `pr-default-branch`, `pr-fork`, `pr-snapshot`, `pr-last-package-version`, `release-release-type`, `release-changelog-path`)
- [ ] change the option name to `release-draft`
- [ ] keep the option name `draft`

Reply:
Let's keep the option `draft` 👍 

If you're so inclined, add `release-create-as-draft`.

Reply:
I'm getting a type error when adding the `alias` key. not sure why but maybe I'll leave it off for now.

```
src/bin/release-please.ts:137:11 - error TS2345: Argument of type '{ describe: string; type: string; default: false; alias: string; }' is not assignable to parameter of type 'YargsOptions'.
  Object literal may only specify known properties, and 'alias' does not exist in type 'YargsOptions'.

137           alias: 'release-create-as-draft',
```